include/modmesh/python/common.hpp

```diff
@@ -300,27 +306,27 @@ WrapBase
         static_assert(!std::is_const<array_reference>::value, "this_array_reference cannot be const");
         using array_type = typename std::remove_reference<array_reference>::type;
 
-        (*this).def_property
-        (
-            name
-          , [&f](wrapped_type & self) -> array_reference { return f(self); }
-          , [&f](wrapped_type & self, py::array_t<typename array_type::value_type> & ndarr)
-            {
-                array_reference this_array = f(self);
-                if (this_array.nbytes() != static_cast<size_t>(ndarr.nbytes()))
+        (*this)
```

Comment:
`clang-format` is particularly limited with lambda, for which I manually change the style.